src/docs/pages/components/variants/Preview.tsx

```diff
@@ -99,7 +99,8 @@ const Preview = (props: Props): JSX.Element => {
       TagController: lazy(() => import('./tag/_TagController')),
       DefaultNavTertiary: lazy(() => import('./navTertiary/DefaultNavTertiary')),
       InputTextController: lazy(() => import('./inputText/_InputTextController')),
-      SelectController: lazy(() => import('./select/_SelectController'))
+      SelectController: lazy(() => import('./select/_SelectController')),
+      TextAreaController: lazy(() => import('./textArea/_TextAreaController'))
```

Comment:
v nitpicky, but textarea is one word, so we can format like `textarea/_TextareaController` 
can we look for casing throughout?